test/ebaze/accounts_test.exs

```diff
@@ -0,0 +1,66 @@
+defmodule Ebaze.AccountsTest do
+  use Ebaze.DataCase
+
+  alias Ebaze.Accounts
+
+  describe "users" do
+    alias Ebaze.Accounts.User
+
+    @valid_attrs %{password: "some password", username: "some username"}
+    @update_attrs %{password: "some updated password", username: "some updated username"}
+    @invalid_attrs %{password: nil, username: nil}
+
+    def user_fixture(attrs \\ %{}) do
+      {:ok, user} =
+        attrs
+        |> Enum.into(@valid_attrs)
+        |> Accounts.create_user()
+
+      user
+    end
+
+    test "list_users/0 returns all users" do
+      user = user_fixture()
+      assert Accounts.list_users() == [user]
+    end
+
+    test "get_user!/1 returns the user with given id" do
+      user = user_fixture()
+      assert Accounts.get_user!(user.id) == user
+    end
+
+    test "create_user/1 with valid data creates a user" do
+      assert {:ok, %User{} = user} = Accounts.create_user(@valid_attrs)
+      assert user.password == "some password"
+      assert user.username == "some username"
+    end
+
+    test "create_user/1 with invalid data returns error changeset" do
+      assert {:error, %Ecto.Changeset{}} = Accounts.create_user(@invalid_attrs)
+    end
+
+    test "update_user/2 with valid data updates the user" do
+      user = user_fixture()
+      assert {:ok, %User{} = user} = Accounts.update_user(user, @update_attrs)
+      assert user.password == "some updated password"
```

Comment:
So it's storing passwords in plain text ATM?

Reply:
updated to implement bcrypt 787628c80a65fed180d839b99132b8871bcbc5bf